kythe/cxx/verifier/assertion_ast.h

```diff
@@ -20,6 +20,8 @@
 #include <algorithm>
 #include <unordered_map>
 #include <vector>
+#include <ctype.h>
+#include <regex>
```

Comment:
Please use RE2.  We already depend on it and the last time I checked all of the implementations of the standard header are terrible. The RE2 header is "re2/re2.h" and the build dep "@com_googlesource_code_re2//:re2"